src/views/TagsList.vue

```diff
@@ -27,9 +27,17 @@ export default {
         .then(data => this.tags = data.data)
         .catch(err => console.log(err));
     }
-  }, 
+  },
   mounted() {
     this.fetchTaglist();
+      const titleEl = document.querySelector('head title');
+      titleEl.textContent = "All Tags";
+
+    const descEl = document.querySelector('head meta[name="description"]');
+    descEl.setAttribute('content', 'See all our tags');
+
+    const robotEl = document.querySelector('head meta[name="robots"]');
+    robotEl.setAttribute('content', 'noindex, nofollow');
```

Comment:
why no follow here?

Reply:
This page lists all the tags. Nothing Else. We are indexing all the pages of individual tags